python/ray/experimental/serve/queues.py

```diff
@@ -207,9 +191,33 @@ def _flush(self):
                     )
                     work.replica_handle._ray_serve_call.remote(request)
 
+    # selects the backend and puts the service queue query to the buffer
+    # different policies will implement different backend selection policies
+    def _flush_service_queue(self):
+        """
+            Expected Implementation:
```

Comment:
Indent level. and please follow the sphinx google code style for comments https://sphinxcontrib-napoleon.readthedocs.io/en/latest/